pkgs/applications/misc/pomodoro/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+let
+  pname = "pomodoro";
+  version = "0.1.0-git";
+in
+rustPlatform.buildRustPackage {
+  inherit pname;
+  inherit version;
+  src = fetchFromGitHub {
+    owner = "SanderJSA";
+    repo = pname;
+    rev = "c833b9551ed0b09e311cdb369cc8226c5b9cac6a";
+    sha256 = "0x5apv5q0dhjrmjrmfzclsgd3ynyjd68llyk2yad8wa9hpanl3b4";
+  };
+
+  cargoSha256 = "0nwf2f6xywirz70kr5xxly5rbp493idfsv8y7bqgnwrp9y63v2q8";
+
+  postPatch = ''
+    substituteInPlace src/main.rs --replace \
+      'Usage pomodoro work_duration pause_duration' \
+      'Usage: pomodoro work_duration_minutes pause_duration_minutes'
+  '';
```

Comment:
If you don't agree with the source code fix it in the source repo not here.

Reply:
Fair point. Removed the patch.